docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -201,20 +236,45 @@ allowed_organizations = 8bab1c86-8fba-33e5-2089-1d1c80ec267d
 
 ### Configure allowed groups
 
-To limit access to authenticated users who are members of one or more groups, set `allowed_groups`
+Azure AD groups can be used to limit user access to Grafana. More info about managing groups in Azure AD can be found [here](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-manage-groups).
+
+To limit access to authenticated users who are members of one or more AzureAD groups, set `allowed_groups`
 to a comma- or space-separated list of group object IDs. You can find object IDs for a specific group on the Azure portal:
 
-1. Go to **Azure Active Directory -> Groups**. If you want to only give access to members of the group `example` with an ID of `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, then set the following:
+1. Go to **Azure Active Directory -> Groups**. If you want to only give access to members of the group `example` with an Object Id of `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, then set the following:
 
    ```
    allowed_groups = 8bab1c86-8fba-33e5-2089-1d1c80ec267d
    ```
 
-1. Verify that [group attributes](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims#configure-the-azure-ad-application-registration-for-group-attributes) is enabled in your Azure AD Application Registration manifest file by navigating to **Azure Portal** > **Azure Active Directory** > **Application Registrations** > **Select Application** -> **Manifest**, and set the following:
+> You can find the Object Id of a group by clicking on the group and then clicking on **Properties**. The object ID is listed under **Object ID**.
 
-   ```
-   "groupMembershipClaims": "ApplicationGroup, SecurityGroup"
-   ```
+1. You can enable adding the [group attribute](https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims#configure-groups-optional-claims) to the tokens in your Azure AD App registration either [from the Azure Portal](#configure-group-membership-claims-on-the-azure-portal) or [from the manifest file](#configure-group-membership-claim-in-the-manifest-file).
```

Comment:
Are we telling the user to do this? The word "can" makes it sound optional.

Reply:
It's required. I replaced "can" with "must";